CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Release History
-## 1.0.0 (2012-11-01)
+
+## 1.1.0 (2023-01-19)
+
+### Behavior Changes
+- `SnowparkSourceCodeExtractionException` would be raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
+- `Session.create_dataframe()` no longer ignores the optional parameter `schema` when `data` is a pandas DataFrame.
+
+### New Features:
+- Added `asc`, `asc_nulls_first`, `asc_nulls_last`, `desc`, `desc_nulls_first`, `desc_nulls_last`, `date_part` in functions.
```

Comment:
Add an "and" before the last item in the list, like this:

Added `asc`, `asc_nulls_first`, `asc_nulls_last`, `desc`, `desc_nulls_first`, `desc_nulls_last`, and `date_part` in functions.